contracts/tatum/custodial/CustodialWalletFactoryV2.sol

```diff
@@ -26,7 +27,11 @@ contract CustodialWalletFactoryV2 {
         exists = wallets[salt] != address(0);
     }
 
-    function getWallets(address owner, uint256[] memory index) public view returns (address[] memory addr, bool[] memory exists, bytes32[] memory salt) {
+    function getWallets(address owner, uint256[] memory index) public view returns (address[] memory _addr, bool[] memory _exists, bytes32[] memory _salt) {
+        address[] memory addr = new address[](index.length); 
```

Comment:
you dont need to define variables here if they are being returned. It's feature of solidity - thats why there were named like that